crates/chia-consensus/src/merkle_set.rs

```diff
@@ -37,13 +45,43 @@ fn hash(ltype: NodeType, rtype: NodeType, left: &[u8; 32], right: &[u8; 32]) ->
     hasher.finalize().into()
 }
 
-const BLANK: [u8; 32] = [
+pub const BLANK: [u8; 32] = [
```

Comment:
Instead of having a public constant for this, you could just use `[0; 32]` anywhere you would use it. This fills a new 32 byte array with zeros.

Reply:
Maybe just not public then, we don't want to clutter the public API with things only relevant to dependents

Reply:
I'll set it to `pub(crate)` so that `merkle_tree.rs` can also use it.